src/cpp/fastdds/builtin/type_lookup_service/TypeLookupReplyListener.cpp

```diff
@@ -53,11 +52,68 @@ TypeLookupReplyListener::~TypeLookupReplyListener()
 {
 }
 
+void TypeLookupReplyListener::check_get_types_reply(
+        const SampleIdentity& request_id,
+        const TypeLookup_getTypes_Out& reply)
+{
+    // Check if we were waiting that reply SampleIdentity
```

Comment:
Please, try to avoid the personal style in comments, here and in other comments throughout this PR.